docs/source/install/tools.rst

```diff
@@ -0,0 +1,91 @@
+==================
+Tools Installation
+==================
+
+Overview
+********
+
+This document describes how to install following PDS Registry tools (command-line applications):
+
+ * **Standalone Harvest** extracts metadata from PDS4 labels and loads extracted metadata into PDS Registry.
+ * **Harvest Client** has to be used with Scalable Harvest server components to submit asynchronous jobs
+   to the server cluster.
+ * **Registry Manager** is a tool to create Registry indices in Elasticsearch and to 
+   manage data and LDDs (data dictionaries).
+ * **Supplementer** can be used to load supplemental metadata into PDS Registry.
+
+
+Standalone Harvest
+******************
+
+Download latest non-snapshot binary release (tar or zip) from https://github.com/NASA-PDS/harvest/releases.
+Extract the **tar** (Linux, Mac) or **zip** (Windows) to a directory without spaces, such as */home/pds*.
+On Linux you can use the following command::
+
+  % tar -xzvf harvest-x.y.z-bin.tar.gz
+
+
+Harvest Client
```

Comment:
Should we use the term "**Harvest Client CLI**" (instead of "**Harvest Client**") here to be consistent with the "**Harvest Client CLI**" term used in the diagram at docs/source/_static/images/registry-harvest.png?

Reply:
All tools have command-line interface (CLI). Let's discuss.

Reply:
I was just thinking about the name we have in https://github.com/NASA-PDS/registry-harvest-cli and docs/source/_static/images/registry-harvest.png, and thought it should be consistent here too.

Reply:
I changed the image. Now it has "Standalone Harvest" and "Harvest Client".

Reply:
That's good